code/frontend/test/SpeechToText.test.ts

```diff
@@ -1,16 +1,36 @@
-import { expect } from "chai";
-import { it } from "mocha";
-import { multiLingualSpeechRecognizer } from "../src/util/SpeechToText.js";
+import { beforeEach, describe, expect, it } from 'vitest'
+import { multiLingualSpeechRecognizer, defaultLanguage } from "../src/util/SpeechToText.js";
+
+const recognizedLanguages = "en-US,es-ES";
 
 describe("SpeechToText", () => {
-  it("creates a speech recognizer with multiple languages", async () => {
+  beforeEach(() => {
+
+  })
+
+  it("creates a speech recognizer with the default language if config not specified", async () => {;
     const token = "token";
     const region = "region";
     const languages = ["en-US", "fr-FR", "de-DE", "it-IT"];
     const recognizer = multiLingualSpeechRecognizer(token, region, languages);
```

Comment:
Is this being overridden by the line below?

Reply:
I'd marked the PR as draft since I was in the middle of making changes. Will submit the PR for review soon.